src/main/java/rs/onako2/iwie/entity/CreakingHeartBlockEntity.java

```diff
@@ -18,13 +30,59 @@ public static void tick(World world, BlockPos pos, BlockState state, CreakingHea
         // check if heart is activated because I forgot it so I didn't want to forget it :)
         if (state.get(CreakingHeartBlock.ACTIVATED)) {
             blockEntity.checkAndSpawnCreaking(world, pos);
+            if (state.getBlock() instanceof CreakingHeartBlock) {
+                if(blockEntity.timing <= 100) {
+                    blockEntity.timing++;
+                }
+            }
+        }
+    }
+    public BlockPos findAirBlockPos() {
+        BlockPos basePos = this.getPos();
+        int[][] offsets = {
+                { 1, 0, 0 },  // 前
```

Comment:
Please write comments in English because it is more widely used than Chinese and I'm not Chinese lol.